master-web/templates/home.jade

```diff
@@ -32,23 +32,23 @@
             a(href="#grid?branch=10.10") 10.10
             | ) will give you a developer-oriented summary of recent buildbot activity.
           li
-            | The 
```

Comment:
We need to be careful about whitespaces into the .html and .jade templates. Some of them may be for display purposes even though are at the end of the line

Reply:
Hmm not sure that I understand this and I am not good with HTML but shouldn't we use the non breaking space then? `&nbsp;`